specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2025-08-01-preview/examples/RedisEnterpriseMigrationGet.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "clusterName": "cache1",
+    "resourceGroupName": "rg1",
+    "api-version": "2025-08-01-preview",
+    "subscriptionId": "e7b5a9d2-6b6a-4d2f-9143-20d9a10f5b8f"
+  },
+  "responses": {
+    "200": {
+      "body": {
```

Comment:
>body [](http://example.com/codeflow?start=7&length=4)

if starting a migration sets these parameters, shouldn't getting the migration be a way to find out what they were set to?

"switchDns": true,
      "skipDataMigration": true